### PR TITLE
Add include_policy directive to policyfile docs

### PR DESCRIPTION
**Pull request by jaym**
Docs for https://github.com/chef/chef-dk/pull/1446

### CODE REVIEW

**Comment by jaym** [comment]
We'll need to update the version based on when we merge

File: chef_master/source/config_rb_policyfile.rst [hunk 1]
```diff
@@ -135,6 +135,20 @@ A ``Policyfile.rb`` file may contain the following settings:
...
 
+``include_policy "NAME", *args``
+  Specify a policyfile lock to be merged with this policy. ChefDK supports pulling this lock from
+  a local file or from chef server.
+
+  ``include_policy "NAME", path: "."`` pulls the lock from ``./NAME.lock.json``.
+
+  ``include_policy "NAME", path: "./foo.lock.json"`` pulls the lock from ``./foo.lock.json``.
+
+  ``include_policy "NAME", policy_revision_id: "abcdabcdabcd", server: "http://example.com"`` pulls the policy ``NAME`` with revision id ``abcdabcdabcd`` from the ``http://example.com`` chef server.
```
Comment (danielsdeleo) [inline chef_master/source/config_rb_policyfile.rst]: Do we fill in the chef server with the default (from config file) if it's not present or did we save that for later?

**Review by danielsdeleo — APPROVED** [review]
Technical content looks good.

File: chef_master/source/config_rb_policyfile.rst (hunk 1 above)
Comment (jaym) [reply chef_master/source/config_rb_policyfile.rst]: We pitched it, never did it from what I can tell.

**Comment by jaym** [comment]
This feature landed in the current channel

**Comment by jaym** [comment]
Looks like this landed in stable

**Pull request merged**